src/gui/systray.cpp

```diff
@@ -197,6 +198,21 @@ bool Systray::isOpen()
     return _isOpen;
 }
 
+QString Systray::windowTitle() const
+{
+    return Theme::instance()->appNameGUI();
+}
+
+bool Systray::useNormalWindow() const
+{
+    if (!isSystemTrayAvailable()) {
+        return true;
+    }
+
+    ConfigFile cfg;
+    return cfg.showMainDialogAsNormalWindow();
```

Comment:
could just be `ConfigFile().showMainDialogAsNormalWindow();` but, no need to change it now